azure/table_azure_postgresql_flexible_server.go

```diff
@@ -47,27 +47,147 @@ func tableAzurePostgreSqlFlexibleServer(_ context.Context) *plugin.Table {
 				Description: "The type of the resource.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "state",
+				Description: "A state of a server that is visible to user.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.State"),
+			},
+			{
+				Name:        "availability_zone",
+				Description: "Availability zone information of the server.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.AvailabilityZone"),
+			},
+			{
+				Name:        "administrator_login",
+				Description: "The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation).",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Properties.AdministratorLogin"),
+			},
+			{
+				Name:        "administrator_login_password",
```

Comment:
@ParthaI I think we should not include this column since it includes sensitive data.